generators/server/templates/dotnetcore/src/Project/Controllers/SwaggerController.cs.ejs

```diff
@@ -0,0 +1,48 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace <%= namespace %>.Controllers
+{
+    [Route("")]
+    [ApiController]
+    public class SwaggerController : ControllerBase
+    {
+        private readonly ILogger<SwaggerController> _log;
+
+        public SwaggerController(ILogger<SwaggerController> log)
+        {
+            _log = log;
+        }
+
+        [HttpGet("swagger-resources")]
+        public ActionResult<IEnumerable<SwaggerResource>> GetSwaggerResources()
+        {
+            _log.LogDebug("REST request to get Swagger Resources");
+            SwaggerResource sr1 = new SwaggerResource() { Name = "<%=pascalizedBaseName%>", Location = "/v2/api-docs" };
+            List<SwaggerResource> swaggerResources = new List<SwaggerResource>();
+            swaggerResources.Add(sr1);
+            return Ok(swaggerResources);
+        }
+    }
+
+    public class SwaggerResource
```

Comment:
Can you extract this class and put it in DTO project please ?

Reply:
Sure!
I Will do it.

Reply:
@nicolas63 Done!